tests/fips/mozilla_nss/firefox_nss.pm

```diff
@@ -69,16 +69,21 @@ sub run {
 
     assert_screen "firefox-device-manager";
 
-    send_key "alt-shift-f";    # Enable FIPS mode
+    # Add condition in FIPS_ENV_MODE & Remove hotkey
+    assert_and_click('firefox-click-enable-fips') if (get_var('FIPS_ENV_MODE'));
+    wait_still_screen 2;
```

Comment:
It is better move this wait into the condition area of 'FIPS_ENV_MODE'. You can decide whether to do it. :)

Reply:
sorry I don't really get it, do you mean write into the same line? do you have some example that I can refer to it? thanks for the comment.

Reply:
After discussing with Lemon, will change like this, 
```
if (get_var('FIPS_ENV_MODE') {
assert_and_click('firefox-click-enable-fips');
wait_still_screen 2;
}
```

Thanks, Lemon.